Server/main.go

```diff
@@ -24,36 +23,42 @@ func main() {
 		log.Fatal("Error loading .env file")
 	}
 
-	// Echo instance
 	e := echo.New()
 
-	// Middleware
 	e.Use(middleware.Logger())
 	e.Use(middleware.Recover())
 
 	// change with dotenv later to remove origins
 	e.Use(middleware.CORSWithConfig(middleware.CORSConfig{
-  		AllowOrigins: []string{"http://localhost:5000"},
+		AllowOrigins:     []string{"http://localhost:5000"},
 		AllowCredentials: true,
 	}))
 
-	// Route => handler
-	rootController.RootController(e)
-
-	e.GET("/api", func(c echo.Context) error {
-		fmt.Printf("/API/ WAS CALLED")
-		usersCollection := Collections.UsersGetCollection()
-		ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
-		defer cancel()
+	sseChannel = entities.SSEChannel{
+		Clients:  make([]chan string, 0),
+		Notifier: make(chan string),
+	}
 
-		cur, _ := usersCollection.Find(ctx, bson.M{})
+	rootController.RootController(e, &sseChannel)
 
-		var users []bson.M
-		err = cur.All(ctx, &users)
+	done := make(chan interface{})
+	defer close(done)
 
-		return c.JSON(200, users)
-	})
+	go broadcaster(done, &sseChannel)
 
-	// Start server
 	e.Logger.Fatal(e.Start(":" + os.Getenv("PORT")))
 }
+
+func broadcaster(done <-chan interface{}, sseChanel *entities.SSEChannel) {
+	fmt.Println("Broadcaster Started.")
+	for {
+		select {
+		case <-done:
+			return
+		case data := <-sseChannel.Notifier:
+			for _, channel := range sseChannel.Clients {
+				channel <- data
+			}
+		}
+	}
+}
```

Comment:
maybe open a seperate folder, Channels and manage everything in there

Reply:
Will do